packages/vm/lib/state/stateManager.ts

```diff
@@ -407,7 +423,10 @@ export default class DefaultStateManager implements StateManager {
     // setup cache checkpointing
     this._cache.revert()
     this._storageTries = {}
-    this._accessedStorage.pop()
+    const lastItem = this._accessedStorage.pop()
+    if (lastItem) {
+      this._accessedStorageReverted.push(lastItem)
```

Comment:
This will degrade performance, right? If we don't want to get the access lists, then we also should not copy these items since it is unnecessary?

Reply:
Ah, yes, you are right. With this openly usable `generateAccessList()` function we might nevertheless want to leave as-is, but haven't thought about this aspect yes.